### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: documentation/example-session.md [hunk 1]
```diff
@@ -7,38 +7,43 @@ const arc = new Arc({
...
+// we get the first return item from the obervable
+const { first } = require('rxjs/operators')
+const daos = await arc.daos().pipe(first()).toPromise()
 
 // you can query for a particular DAO
 arc.daos({ name: 'mydao'})
 
-const dao = await arc.dao(daos[0].address)
+// or if you know the address, just create a new DAO object like this:
+const dao = arc.dao(daos[0].address)
```
Comment (ben-kaufman) [inline documentation/example-session.md]: Why did you remove the `await`?

File: documentation/example-session.md [hunk 2]
```diff
@@ -7,38 +7,43 @@ const arc = new Arc({
...
-const { name }  = await dao.state.toPromise()
+const state  = await dao.state.pipe(first()).toPromise()
+// the state contains information such as the address, name or nativeToken of the DAO
+const { address, name, token }  = state
 
 // proposals
 // get a Observable of proposals that are active but not boosted
-const proposals = await dao.proposals({ active: true, boosted: false }).toPromise()
+const proposalsObservable = await dao.proposals({ active: true, boosted: false })
+const currentProposals = proposalsObservable.pipe(first()).toPromise()
```
Comment (ben-kaufman) [inline documentation/example-session.md]: Why do you use `first()` here?

File: documentation/example-session.md (hunk 1 above)
Comment (jellegerbrandy) [reply documentation/example-session.md]: because it is not asynchronous, so it is not needed

File: documentation/example-session.md (hunk 2 above)
Comment (jellegerbrandy) [reply documentation/example-session.md]: `observer.toPromose()` resolves when the observer is complete. 
`proposalsObservable` is never complete - it is always looking for newly updated proposals
So what i do is create a new observable that just emits the first value and then completes (this is what `first()` does), and then check that promise...

Comment (ben-kaufman) [reply documentation/example-session.md]: Oh ok, I misunderstood first, thought it gives just the first element in the list of proposals...
